ssntp/client.go

```diff
@@ -368,7 +368,7 @@ func (client *Client) sendCommand(cmd Command, payload []byte, trace *TraceConfi
 	client.status.Lock()
 	if client.status.status == ssntpClosed {
 		client.status.Unlock()
-		return -1, fmt.Errorf("Client not connected")
+		return -1, fmt.Errorf("sendCommand: Client not connected")
 	}
 	client.status.Unlock()
 
```

Comment:
Name and uuid, if we're actually thinking somebody will track it back to the running thing that got missed.  My inclination is to think nobody's going to see that or care.  Maybe the message should be more like "sendCommand: discarding command to disconnected client"?